ranger/config/rc.conf

```diff
@@ -593,15 +593,19 @@ map zf    console filter%space
 copymap zf zz
 
 # Filter stack
-map .n console filter_stack add name%space
-map .m console filter_stack add mime%space
 map .d filter_stack add type d
 map .f filter_stack add type f
 map .l filter_stack add type l
+map .m console filter_stack add mime%space
+map .n console filter_stack add name%space
+map .# console filter_stack add hash%space
+# alternatively .: and .;?
```

Comment:
I like the mnemonics of <kbd>."</kbd> and <kbd>.'</kbd>, let's go with them and remove this comment.

Reply:
I got `"` from the "ditto" symbol and `'` was a nice counterpoint.
Though `:` looks more like "a list of duplicates" while `;` looks like "a list of non-duplicates."

We could also map all of the above but I'll remove the comment for now.